### DISCUSSION TITLE
Lora vs checkpoint

### DISCUSSION THREAD

**Discussion by bejaranoo** [TheLastBen/fast-stable-diffusion#2697] [Q&A]
I was playing around with this guide (https://civitai.com/models/22530) and made a comparison, Lora vs checkpoint, both using same dataset of images and tested in with same prompts. 

I got to say the Lora file only weights 18mb and manages to accomplish the same results with the difference that I have to first generate a good image with a random person and then use inpainting to paste the trained face Lora, but this because my data set was trained almost entirely on close up photos/selfies. Also using too many Loras on one prompt can cause deformed images, but nothing playing with weights cannot fix.

All this got me wondering why isn't Lora training for custom faces more popular, it seems to me the better option, or am I missing some advantages of having a trained checkpoint/model?

**Comment by TheLastBen** [comment]
Lora for a face makes it difficult to keep consistency of the face in complicated prompts
